circuit/src/execution-delivery/src/tests.rs

```diff
@@ -1388,3 +1393,61 @@ fn error_if_incorrect_escrow_is_submitted() {
         assert!(submitter.is_err());
     });
 }
+
+#[test]
+fn test_submit_composable_exec_order() {
+    let io_schedule = b"component1;".to_vec();
+
+    let components = vec![Compose {
+        name: b"component1".to_vec(),
+        code_txt: r#""#.as_bytes().to_vec(),
```

Comment:
Must pass a valid contract WASM smart contract code here, since the `from_compose` converts it into temp contract. 
```rust
   20     pub fn from_compose(
   19         compose: Compose<AccountId, BalanceOf>,
   18         action_descriptions: Vec<ContractActionDesc<Hash, ChainId, AccountId>>,
   17         author: AccountId,
   16         author_fees_per_single_use: Option<BalanceOf>,
   15         abi: Option<Vec<u8>>,
   14         info: RawAliveContractInfo<Hash, BalanceOf, BlockNumber>,
   13     ) -> RegistryContract<Hash, AccountId, BalanceOf, BlockNumber> {
   12         RegistryContract::new(
   11             compose.code_txt,
   10             compose.bytes,
    9             author,
    8             author_fees_per_single_use,
    7             abi,
    6             action_descriptions,
    5             info,
    4         )
    3     }
   ```
   See [pallet-contracts fixtures examples](https://github.com/paritytech/substrate/tree/master/frame/contracts/fixtures)